third_party/closure-library/compile.sh

```diff
@@ -28,9 +28,11 @@ java -jar $1 \
     --js "$2/goog/reflect/reflect.js" \
     --js "$2/goog/string/string.js" \
     --js_output_file "sha384-generated.js" \
-    --output_wrapper "/* Generated from closure library commit $GIT_COMMIT
-*/%output%;exports.base64=function base64(input) { return ampBase64(input) };
```

Comment:
do we just not use base64 anywhere?

Reply:
Yea, we don't even export `ampBase64` in sha384.js.